rest-api-spec/src/main/resources/rest-api-spec/test/search/340_flattened.yml

```diff
@@ -1,9 +1,10 @@
----
-"Test exists query on flattened object field":
+setup:
   - skip:
-      version: " - 7.2.99"
-      reason: "Flattened fields were implemented in 7.3."
+      version: " - 7.99.99"
```

Comment:
Version change because some nodes may now have x-pack installed?

Reply:
These core REST tests generally do not run with x-pack installed. So I bumped the version here, and plan to adjust to cover 7.12 once it's backported.